lib/resty/mlcache.lua

```diff
@@ -286,15 +311,16 @@ function _M:get(key, opts, cb, ...)
 
     -- opts validation
 
+    local lru
     local ttl
     local neg_ttl
 
-    if opts then
+    if opts ~= nil then
```

Comment:
unless theres an explicit reason for this change, it seems this change in type checking belongs in a separate commit.